GIT_AND_GITHUB.md

```diff
@@ -89,6 +77,13 @@ discuss other alternatives. Of course, this means you are putting yourself at ri
 of sharing your knowledge and/or learning some new stuff, so please be careful not
 to end up being even more awesome than you are.
 
+If everything is fine feel free to merge the pull request. Avoid merging before receiving
```

Comment:
We might refer to "the one making the PR" as "the submitter" as we do to "the reviewer" so it becomes more clear that here we go back to the role as "the submitter".

So here it could say:

"As the submitter you can feel free to merge the pull request if everything is fine"